server/app.js

```diff
@@ -56,6 +60,13 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
+// POST request for uploading to Express server then pushing to S3 Bucket
+app.post('/profilePhotos', upload.single('profilePhoto'), async (req, res) => {
+  const file = req.file;
+  const result = await uploadPhoto(file);
```

Comment:
You need to confirm that upload was successful before returning `success` to the client.